src/actions/add-on-types-actions.js

```
@@ -0,0 +1,183 @@
+import T from "i18n-react/dist/i18n-react";
+import {
+  getRequest,
+  putRequest,
+  postRequest,
+  deleteRequest,
+  createAction,
+  stopLoading,
+  startLoading,
+  escapeFilterValue
+} from "openstack-uicore-foundation/lib/utils/actions";
+import { snackbarErrorHandler, snackbarSuccessHandler } from "./base-actions";
```

ESLint

[error] snackbarErrorHandler not found in './base-actions'

(import/named)